docs/content/docs/getting-started/contributing.md

```diff
@@ -5,39 +5,44 @@ weight: 50
    {{< details "Get to know general workflow for VCR tests" >}}
 
-      1. You submit your change.
+      1. Submit your change.
       1. The recorded tests are ran against your changes by the `modular-magician`. Tests will fail if:
          1. Your PR has changed the HTTP request values sent by the provider
          1. Your PR does not change the HTTP request values, but fails on the values returned in an old recording
          1. The recordings are out of sync with the merge-base of your PR, and an unrelated contributor's change has caused a false positive
       1. The `modular-magician` will leave a message indicating the number of passing and failing VCR tests. If there is a failure, the `modular-magician` user will leave a message indicating the "`Triggering VCR tests in RECORDING mode for the following tests that failed during VCR:`" marking which tests failed.
-         1. If a test does not appear related, it probably isn't!
+         1. If a test does not appear related to your PR, it probably isn't!
       1. The `modular-magician` will kick off a second test run targeting only the failed tests, this time hitting the live GCP APIs. If there are tests that fail at this point, a message stating `Tests failed during RECORDING mode:` will be left indicating the tests.
          1. If a test that appears to be related to your change has failed here, it's likely your change has introduced an issue. You can view the debug logs for the test by clicking the "view" link beside the test case to attempt to debug what's going wrong.
-         1. If a test appears to be completely unrelated has failed, it's possible that a GCP API has changed in a way that broke the provider or our environment capped on a quota.
+         1. If a test that appears to be completely unrelated has failed, it's possible that a GCP API has changed in a way that broke the provider or our environment capped on a quota.
    {{< /details >}}
 
    Where possible, take a look at the logs and see if you can figure out what needs to be fixed related to your change.
-   The false positive rate on these tests is extremely high between changes in the API, Cloud Build bugs, and eventual consistency issues in test recordings so we don't expect contributors to wholly interpret the results- that's the responsibility of your reviewer.
-1. If your assigned reviewers does not reply / review within a week, gently ping them on github.
-1. After your PR is merged, it will be released to customers in around a week or two.
+   The false positive rate on these tests is extremely high between changes in the API, Cloud Build bugs, and eventual consistency issues in test recordings so we don't expect contributors to wholly interpret the results — that's the responsibility of your reviewer.
+1. If your assigned reviewer does not reply / review within a week, gently ping them on GitHub.
+1. After your PR is merged, it will be released to customers in one to two weeks.
```

Comment:
I would leave the "around" qualifier here. We can't guarantee timelines, and I think we want to make sure that expectations are set appropriately, especially because release dates can be a sticking point for customers.